domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimDocumentCollection.java

```diff
@@ -33,4 +33,12 @@ public void addStaffUploadedDocument(ClaimDocument claimDocument) {
             .filter(claimDocument -> claimDocument.getDocumentType().equals(claimDocumentType))
             .findFirst();
     }
+
+    public Optional<ScannedDocument> getScannedDocument(ScannedDocumentType scannedDocumentType,
+                                                        ScannedDocumentSubtype subtype) {
+        return scannedDocuments.stream()
+            .filter(scannedDocument -> scannedDocument.getDocumentType().equals(scannedDocumentType))
+            .filter(scannedDocument -> scannedDocument.getSubtype().equals(subtype.value))
+            .findFirst();
```

Comment:
There may be multiple forms of the same type and subtype. We should return the most recent one. ScannedDocuments have a deliveryDate so we can determine which is the most recent

Reply:
Done in next commit